diff -ur libmpd-11.8.17/src/Makefile.am libmpd-11.8.17/src/Makefile.am
--- libmpd-11.8.17/src/Makefile.am	2011-08-17 20:31:47.000000000 +0800
+++ libmpd-11.8.17/src/Makefile.am	2023-06-19 21:20:45.149253459 +0800
@@ -1,7 +1,7 @@
 INCLUDES = 				\
 	-I.				\
 	-I$(top_srcdir)			\
-	@EXTRA_CFLAGS@ -Wall -Wmissing-declarations 
+	@EXTRA_CFLAGS@ -Wall -Wmissing-declarations -Wno-int-conversion
 AM_CFLAGS=@glib_CFLAGS@
 LIBS=@glib_LIBS@
 
diff -ur libmpd-11.8.17/src/Makefile.in libmpd-11.8.17/src/Makefile.in
--- libmpd-11.8.17/src/Makefile.in	2011-08-17 20:32:05.000000000 +0800
+++ libmpd-11.8.17/src/Makefile.in	2023-06-19 21:20:45.149253459 +0800
@@ -224,7 +224,7 @@
 INCLUDES = \
 	-I.				\
 	-I$(top_srcdir)			\
-	@EXTRA_CFLAGS@ -Wall -Wmissing-declarations 
+	@EXTRA_CFLAGS@ -Wall -Wmissing-declarations -Wno-int-conversion
 
 AM_CFLAGS = @glib_CFLAGS@
 lib_LTLIBRARIES = libmpd.la
